master.jinja2

```diff
@@ -10,7 +10,10 @@
 {% set test_timeout = test_timeout|default(60) %}
 {% set TEST_DEFINITIONS_REPOSITORY = TEST_DEFINITIONS_REPOSITORY|default("https://github.com/Linaro/test-definitions.git") %}
 
-{% set job_timeout = lxc_deploy_timeout + lxc_boot_timeout + lxc_install_fastboot_timeout + target_deploy_timeout + TARGET_BOOT_TIMEOUT + test_timeout %}
+{% set job_timeout = job_timeout|default(lxc_deploy_timeout + lxc_boot_timeout + lxc_install_fastboot_timeout + target_deploy_timeout + TARGET_BOOT_TIMEOUT + test_timeout) %}
```

Comment:
Why is this needed ?
Can't we extend this so it will work for android too instead of just overwrite it?

Reply:
first, for this master.jinja2, I think it's better to not control too specific, so that others cases that don't need to think too much about the variables that it actually won't use any more. Like for the android case, the lxc_xxx_timeout variables.

Reply:
anyway, I updated to not change here to avoid affects on the existing files as possible as I can.